snap_format.pm

```diff
@@ -1,5 +1,36 @@
 # this module writes out the chandra snapshot
 
+use JSON;
+
+sub write_json {
+    my %h = @_;
+
+    my %status_assignments = (B => 'unchecked',
+                              G => 'nominal',
+                              Y => 'caution',
+                              R => 'warning',
+                              S => 'stale15min',
+                              I => 'invalid');
+
+    my %new_hash;
+
+    for my $msid (keys %h) {
+        my $status = 'unchecked';
+        for my $stat (keys %status_assignments) {
+            if ($h{$msid}[2] eq $stat) {
+                $status = $status_assignments{$stat};
+            }
+        }
+
+        $new_hash{$msid} = {'time' => $h{$msid}[0],
```

Comment:
Output this time as a string in the standard YYYY:DOY:HH:MM:SS.sss format so there is no ambiguity in interpretation by downstream apps.